src/controllers/appoint.director.submitted.controller.ts

```diff
@@ -1,10 +1,33 @@
 import { NextFunction, Request, Response } from "express";
 import { Templates } from "../types/template.paths";
+import { urlUtils } from "../utils/url";
+import { Session } from "@companieshouse/node-session-handler";
+import { getCompanyProfile } from "../services/company.profile.service";
+import { CompanyProfile } from "@companieshouse/api-sdk-node/dist/services/company-profile/types";
+import { getOfficerFiling } from "../services/officer.filing.service";
+import { formatTitleCase } from "../services/confirm.company.service";
+import { retrieveDirectorNameFromFiling } from "../utils/format";
+import { toReadableFormat } from "../utils/date";
+import { CREATE_TRANSACTION_PATH } from "../types/page.urls";
+import { OfficerFiling } from "@companieshouse/api-sdk-node/dist/services/officer-filing";
 
 export const get = async (req: Request, res: Response, next: NextFunction) => {
   try {
+    const companyNumber = urlUtils.getCompanyNumberFromRequestParams(req);
+    const transactionId = urlUtils.getTransactionIdFromRequestParams(req);
+    const submissionId = urlUtils.getSubmissionIdFromRequestParams(req);
+    const session: Session = req.session as Session;
+    const companyProfile: CompanyProfile = await getCompanyProfile(companyNumber);
+    const officerFiling: OfficerFiling = await getOfficerFiling(session, transactionId, submissionId);
+
     return res.render(Templates.APPOINT_DIRECTOR_SUBMITTED, {
```

Comment:
I can see back link on the prototype but not here. I suppose it was agreed to remove?

Reply:
The prototype does not align with AC written, after discussing with the Product team we went with the decision to match the TM01 implementation and ACs written with the old prototype.  
The AC and TM01 don't have a back button hence I have not implemented it. 

Any new changes if they may come will be addressed in the snag tickets. 
https://companieshouse.atlassian.net/browse/DACT-679?focusedCommentId=226929